packages/lesspass-entropy/src/index.js

```diff
@@ -1,4 +1,5 @@
 const pbkdf2 = require("./pbkdf2");
+const crypto = require("crypto");
```

Comment:
same
We can't import the node module directly here, browserify crypto node module are too slow on the browser.